config/routes.rb

```diff
@@ -1,3 +1,5 @@
 Rails.application.routes.draw do
+  root to: 'keywords#index'
```

Comment:
How about adding a new line after it? So that the `root route` is separated from other routes?
```suggestion
  root to: 'keywords#index'
  
```